exercises/error-handling/error_handling_test.go

```diff
@@ -11,7 +11,7 @@ import (
 // as defined by the expectations of this test suite.
 //
 // Also define a testVersion with a value that matches
-// the internal targetTestVersion here.
+// the targetTestVersion here.
```

Comment:
Got too excited I guess, sorry! I guess I don't need to get to excited since x-api can't update without you anyway. Had I looked at this PR I might have known to wait on the other one. It's like "read the entire test before answering any questions", or something.